src/Conf.php

```diff
@@ -29,9 +29,9 @@ class Conf
     /**
      * Create new instance of Conf class.
      */
-    public function __construct()
+    public function __construct($filepath)
     {
-        $this->file = storage_path('app/conf.json');
+        $this->file = ($filepath) ? $filepath : storage_path('app/conf.json');
```

Comment:
Not really sure I understand why you say this change won't work... I've already tested and it's what I need... I first tried to extend Conf.php and overwrite the constructor with the path I need but then I saw that the properties are private. 
I add the library via composer and need a diferent filepath for my configuration. Even if I would add a config file I'd still need to add that in storage everytime I'm checking out my project...
Can you at least make the properties protected and not private?